tests/Frontend/CoreSpec.hs

```diff
@@ -72,7 +72,9 @@ spec = do
         , formTest (arb :: Gen PageAdminSettingsQuorum)
         , formTest (arb :: Gen PageAdminSettingsFreeze)
         , formTest (arb :: Gen PageAdminSettingsEventsProtocol)
-        , formTest (adminEditUserGen :: Gen AdminEditUser)
+        , formTest (AdminEditUser <$> arb <*> pure schoolClasses)
+                  -- FIXME: Generate the payload based on the AdminEditUser type
```

Comment:
ok, i think i understood it.  it would have been much easier if it had been attached to the code location where it actually can be fixed, namely the `ArbFormPagePayload` instance.  fixed in cf133f5.